npe2/_command_registry.py

```diff
@@ -2,60 +2,106 @@
 from __future__ import annotations
 
 from functools import partial
-from typing import Any, Callable, Dict
+from typing import Any, Callable, Dict, Optional, Union
 
 PDisposable = Callable[[], None]
+import re
+from dataclasses import dataclass
+from importlib import import_module
 
 from psygnal import Signal
 
+from .manifest.commands import _dotted_name
+
+_dotted_name_pattrn = re.compile(_dotted_name)
```

Comment:
spelling? made a suggestion to fix here and in the one other place below.
```suggestion
_dotted_name_pattern = re.compile(_dotted_name)
```